### DISCUSSION TITLE
PENTAHO-PDI or CARTE on a DOLPHINSCHEDULER worker node

### DISCUSSION THREAD

**Discussion by Arturo-Penas-Rial** [apache/dolphinscheduler#12342] [Q&A]
Hi guys,

I need to implement PENTAHO-PDI or CARTE on a DOLPHINSCHEDULER worker node. Is there a Dockerfile that I can use or a documentation that guides me for its implementation?
 
I'm new to DOLPHIN SCHEDULER, it's a fantastic product, but I can't find the documentation to integrate PENTAHO PDI. I will appreciate your help.

**Comment by DarkAssassinator** [comment]
hi @Arturo-Penas-Rial, currect DS did not support PENTAHO-PDI task. May u can raise a PR or just send this feature to email list or issues.

**Reply by Arturo-Penas-Rial** [reply]
Thanks for your answer. I guess it's possible to install PENTAHO-PDI on a DS worker node, and then invoke an ETL execution via remote shell or HTTP call (in case the worker has PENTAHO-CARTE)

**Comment by ayeah** [comment]
You can start carte service and call API via DS through HTTP, simple `executeTrans` works and that's enough for me 
https://help.hitachivantara.com/Documentation/Pentaho/8.3/Developer_center/REST_API_Reference/Carte
